src/mpc-hc/MainFrm.cpp

```diff
@@ -13195,12 +13196,12 @@ void CMainFrame::OpenSetupWindowTitle(bool reset /*= false*/)
                     }
                 }
 
+                if (has_title & title != GetFileName()) m_current_rfe.title = title;

```

Comment:
Maybe add `CString fn = GetFileName()` so that GetFileName() does not need to be called twice.